tests/test_utils.py

```diff
@@ -18,6 +18,47 @@ def __eq__(self, *args):
 
 class MakeRequestTestCase(unittest.TestCase):
 
+    @patch('urllib3.ProxyManager')
+    def test_makes_request_with_proxy(self, mock_manager):
```

Comment:
I would place `test_makes_request_with_proxy` after `test_makes_request` in the class, as it's a less generic case.